eclipse-scout-core/src/widget/FilterSupport.js

```diff
@@ -164,6 +162,16 @@ export default class FilterSupport extends WidgetSupport {
     this._filterField.keyStrokeContext.registerKeyStroke(this._exitFilterFieldKeyStroke);
   }
 
+  _updateFilterFieldBackgroundColor() {
+    let color = styles.getFirstOpaqueBackgroundColor(this._filterField.$container),
```

Comment:
Please check if filterField is still rendered. It may have been removed in the mean time.